packages/blockchain-wallet-v4-frontend/src/components/Balances/selectors.ts

```diff
@@ -107,6 +107,15 @@ export const getEthBalance = createDeepEqualSelector(
   }
 )
 
+export const getEthBalances = createDeepEqualSelector(
+  [getEthNonCustodialBalance, getCoinCustodialBalance('ETH')],
+  (balancesR, custodialBalanceR) => {
+    const custodialBalance = custodialBalanceR.getOrElse(0)
+
+    return Remote.of([new BigNumber(balancesR.getOrElse(new BigNumber(0))), custodialBalance])
```

Comment:
Does it need to be a `BigNumber` inside the `getOrElse`?

Can it be just

```suggestion
    return Remote.of([new BigNumber(balancesR.getOrElse(0)), custodialBalance])
```